packages/@react-types/dialog/src/index.d.ts

```diff
@@ -36,10 +36,17 @@ export interface SpectrumBaseDialogProps extends HTMLAttributes<HTMLElement> {
 }
 
 export interface SpectrumDialogProps extends DOMProps, StyleProps {
+  /** The contents of the dialog. */
   children: ReactNode,
+  /**
+   * Object containing the props that each Dialog child receives,
+   * see [slots](./Slots.html).
+   */
   slots?: Slots,
+  /** How wide the dialog should be or make fullscreen. */
```

Comment:
Instead of using the word 'wide' I think we should be talking about the 'width', maybe using the word 'option' as well.